runtime/lua/vim/_buf.lua

```diff
@@ -0,0 +1,20 @@
+local M = {}
+
+--- Adds one or more blank lines above or below the cursor.
+--- @param above? boolean Place blank line(s) above the cursor
+local function add_blank(above)
+  local offset = above and 1 or 0
+  local repeated = vim.fn['repeat']({ '' }, vim.v.count1)
+  local linenr = vim.api.nvim_win_get_cursor(0)[1]
+  vim.api.nvim_buf_set_lines(0, linenr - offset, linenr - offset, true, repeated)
+end
+
+function M.space_above()
```

Comment:
Add the TODO comments to this file too. Once the TODO is resolved these should be inlined into `_defaults.lua` so let's make that clear to future readers.

Reply:
Yes, sorry, that makes sense. Done.